### PR TITLE
feat: Add rain based on points amount.

### PR DESCRIPTION
**Pull request by KararTY**
Preview:
image

#Hacktoberfest

### CODE REVIEW

File: changelog.html [hunk 1]
```diff
@@ -68,5 +68,9 @@
 <br>
 v1.1.1<br>
 *Modernized code style<br>
+<br>
+v1.2 - The Rain Update<br>
+*Added switch for rain feature<br>
```
Comment (tonkku107) [inline changelog.html:73]: Would make more sense if these two were in reverse order

File: index.html [hunk 2]
```diff
@@ -75,10 +76,11 @@
         <button id="save">Save</button>
         <button id="load">Load</button>
         <button id="reset">Reset</button>
-        <button id="theme">Theme</button><br>
+        <button id="theme">Theme</button>
+        <button id="rain">Rain</button><br>
         <small style="display: none; top: 0px" id="msg"></small>
         <div id="footer">
-        	<small style="top: 0px;">Based on <a href="http://kararty.itch.io/poopclicker">KararTY's poopclicker</a> • Made by tonkku107 • <a href="https://github.com/tonkku107/poop-clicker">Fork on GitHub</a> • Current version: 1.1.1 <a target="_blank" href="changelog.html">(Changelog)</a></small>
```
Comment (tonkku107) [inline index.html:81]: You don't want credit anymore? :(

File: script.js [hunk 3]
```diff
@@ -166,6 +175,7 @@ $(document).ready(() => {
     if (localStorage.getItem('poopsave') !== null) {
       const ret = JSON.parse(atob(localStorage.getItem('poopsave')));
       dark = ret.dark || false;
+      rain = ret.rain || true;
```
Comment (tonkku107) [inline script.js]: This would always result in true

File: script.js [hunk 4]
```diff
@@ -210,10 +221,84 @@ $(document).ready(() => {
...
+  const makeItRain = () => {
+    if (rain) {
+      $('#rainCanvas').show();
 
-  // Show everything if already played
-  if (clickedOnce === 1) {
+      const canvas = $('#rainCanvas').get(0);
+
+      canvas.height = window.innerHeight;
+      canvas.width = 1920;
```
Comment (tonkku107) [inline script.js]: What about wider monitors?

File: script.js [hunk 5]
```diff
@@ -210,10 +221,84 @@ $(document).ready(() => {
...
+
+        /**
+         * If all particles are removed, but poop is more than 1,
+         * allow particles to refresh by adding 1 particle.
+         */
+        if (particles.length === 0 && poop > 0) {
+          particles = [null];
+        }
+
+        for (let index = 0; index < particles.length; index++) {
```
Comment (tonkku107) [inline script.js]: could be `for (const particle of particles)`

File: script.js [hunk 6]
```diff
@@ -210,10 +221,84 @@ $(document).ready(() => {
...
+          if (particles[index] == null) {
```
Comment (tonkku107) [inline script.js]: `if (!particles[index])` or `if (!particle)`

**Review by tonkku107 — CHANGES_REQUESTED** [review]

File: index.html (hunk 2 above)
Comment (KararTY) [reply index.html:81]: I'm a soon-to-be project contributor now, that's enough credit.

File: script.js (hunk 3 above)
Comment (tonkku107) [reply script.js]: Are you sure you want to default it to false now?

**Review by tonkku107 — APPROVED** [review]

**Pull request merged**